src/index.js

```diff
@@ -96,27 +97,50 @@ function validateForm() {
     if (!moment(selectedDate.value)._isValid || moment(selectedDate.value).isBefore(moment(Date.now()))) {
         dateError.classList.remove('hidden');
         validated = false;
-    } else {
-        validated = true;
+    // } else {
+    //     validated = true;
     }
-    if (isNaN(selectedDuration.value)) {
+    if (!selectedDuration.value || typeof(+selectedDuration.value) !== 'number') {
```

Comment:
What's the benefit fo `typeof(+selectedDuration.value) !== 'number')` over `isNaN`?

Reply:
I think I was struggling with isNaN(). I'm not as familiar with it, so I went with something I understood better. Then I realized I needed to turn selectedDuration.value into a number, hence the `+`.